int64 age
string test
string information
string accuracy
string type
---
float64 p_choice




